drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -222,15 +227,13 @@ def client_factory(create: true) # rubocop:disable Lint/UnusedMethodArgument
     end
   end
 
-  # Common HUD Assessment-related attributes
+  # Common HUD Assessment-related attributes. These always have enrollments so enrollment_factory *should* exist
   def common_attributes
     data_collection_stage = owner.data_collection_stage if owner.respond_to?(:data_collection_stage)
-    personal_id = owner.personal_id if owner.respond_to?(:personal_id)
-    information_date = owner.information_date if owner.respond_to?(:information_date)
+    personal_id = enrollment_factory(create: false)&.personal_id
```

Comment:
I tested the removal of `information_date` here with an intake. It looks ok to me, the disability records still collect information date from [here](https://github.com/greenriver/hmis-warehouse/blob/f039ec719577065e8d3c7c0987b0ee3e3a131c40/drivers/hmis/app/models/hmis/hud/processors/base.rb#L37). I think @gigxz you are right that this is legacy. But if you are nervous to remove it I can revert the change.

Reply:
this looks like a good clean up to me, thanks